src/app/rpmostree-clientlib.cxx

```diff
@@ -255,6 +261,19 @@ new_client_connection()
   return std::make_unique<ClientConnection>(conn, bus_type);
 }
 
+// Connect to a transaction DBus and monitor its progress synchronously,
+// printing output to stdout.  Add a signal handler for SIGINT to cancel
+// the transaction.
+void 
+ClientConnection::transaction_connect_progress(const rust::Str address) const
```

Comment:
Feels like `sync` should be in there. Maybe just matching the underlying lingo; e.g. `ClientConnection::transaction_get_response_sync()`?